packages/theme/src/button-group/vars.less

```
@@ -79,4 +79,12 @@
   --tv-ButtonGroup-responsive-item-btn-disabled-text-color: var(--tv-color-text-placeholder, #808080);
   --tv-ButtonGroup-responsive-item-icon-bg-color: var(--tv-color-warn-bg-light, #ffebd1);
   --tv-ButtonGroup-responsive-item-sup-icon-radius: var(--tv-border-radius-md, 6px);
+  // mini 尺寸
```

Stylelint (17.5.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)